tests/install_swiftpm_dependencies/test_scripts/run_tests_with_xcodebuild.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -eu
+
+set -o pipefail
+
+echo "--- :xcode: Run tests to verify packages have been fetched and are available"
```

Comment:
Mhh ok I had some luck with using fake `HTTP_PROXY` as workaround…
```
$ HTTP_PROXY=test-expects-no-network-access HTTPS_PROXY=test-expects-no-network-access xcodebuild -workspace podcasts.xcworkspace -scheme pocketcasts -resolvePackageDependencies

xcodebuild: error: Could not resolve package dependencies:
  Couldn’t fetch updates from remote repositories:
    fatal: unable to access 'https://github.com/dagronf/SwiftSubtitles/': Could not resolve proxy: test-expects-no-network-access
  binary target 'absl' could not be mapped to an artifact with expected name 'absl'
  binary target 'FirebaseFirestoreInternal' could not be mapped to an artifact with expected name 'FirebaseFirestoreInternal'
  binary target 'GoogleAppMeasurementOnDeviceConversion' could not be mapped to an artifact with expected name 'GoogleAppMeasurementOnDeviceConversion'
  binary target 'openssl_grpc' could not be mapped to an artifact with expected name 'openssl_grpc'
  binary target 'Sentry' could not be mapped to an artifact with expected name 'Sentry'
  binary target 'Clibsodium' could not be mapped to an artifact with expected name 'Clibsodium'
```
But… that also fails if the `~/Library/Cache` exists while the DerivedData doesn't exist yet (which is the case we want to test to ensure it uses the cache instead of network…), I think because even if it uses the cache for repository cloning, it still expects to check with the remote to validate the locale cache is not stale.

So overall not sure how we can make the test ensure cache is used and not skipped 🤔

Reply:
Ok after taking a break and thinking back on this, I realized that the goal of those intregration tests was not really to validate that the subsequent `xcodebuild -resolvePackageDependencies` calls would use the cache instead of the network, but rather than the call to `install_swiftpm_dependencies` was using the right `Package.resolved` and especially the right one for the right workspace/project/package based on the input `--flags` used.

So maybe it's all ok after all and we can disregard my whole comments above…